scripts/gulp-start-gae-devserver.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
Actually no I have tried to use it and it only supports dev_appserver.py, which means ever developer have to install appengine and and export path.
here is the error
``` Invalid action /home/barnie/Software/openSource/oppia_tools/google_appengine_1.9.19/google_appengine/dev_appserver.py. Supported actions are dev_appserver.py and appcfg.py
```